e2e_samples/parking_sensors/src/ddo_transform/.devcontainer/Dockerfile

```
@@ -2,22 +2,30 @@
 # Copyright (c) Microsoft Corporation. All rights reserved.
 # Licensed under the MIT License. See LICENSE in the project root for license information.
 #-----------------------------------------------------------------------------------------
-FROM python:3.7
+
+# We are freezing at 3.11 because the build processing is using setup.py and setuptools which is deprecated in 3.13
+FROM python:3.11
```

Was this working for MAC @raafatzarka? On the main container we needed to refer to the architecture like below. As we need to stay at 3.11 I'm suggesting to re-write as below:
```suggestion
FROM --platform=linux/amd64 python:3.11
```